pkgs/applications/editors/eclipse/plugins.nix

```diff
@@ -15,80 +34,123 @@ rec {
 
       passthru = {
         isEclipsePlugin = true;
+        dependencies = [];
```

Comment:
Dependencies are just a list of other plugins (which are of course local p2 repos). When we install our plugins these repos are just added to the set known to the resolver. Not sure if this is an appropriate use of passthru, or if there is a better way to express any of this.